docs/axelar/intro-to-axelar.md

```diff
@@ -0,0 +1,103 @@
+# Axelar Integration
+
+The Axelar-XRPL integration enables messages to pass between the XRPL and other Axelar-supported blockchains. This integration enables the transfer of XRP and XRPL-issued tokens to other chains and back; this also works for briding non-XRPL tokens from their native chains to XRPL.
```

Comment:
Here is a diagram that may be helpful. Let me know if you need it changed in any way to make it easier to understand
![XRPL EVM Axelar Topology2](https://github.com/ripple/opensource.ripple.com/assets/6089916/d200b117-9777-428d-9c0b-b8adc4e8a2cb)